3rdparty/our-vcpkg-dependencies/vcpkg-arm64-osx-dependencies

```diff
@@ -0,0 +1,9 @@
+--triplet
+arm64-osx
```

Comment:
I can't comment on the discord binary, so commenting here - could you add instructions on how can we build it ourselves?

Reply:
Is there an official arm64-osx Discord application - even if there is I am not sure there is a build of the [Discord-RP library](https://github.com/discord/discord-rpc) that can connect to it. It is pretty much unchanged since the Discord people switched over to their SDK (which I think they have also abandoned). Since the latter is not licenced in a way we could use it we've stuck with an out-of-date build of this MIT licenced library instead.

(BTW there has been a later commit that modifies one of the header files https://github.com/discord/discord-rpc/pull/306 but since we don't use the feature it added - about "party privacy" - we've not bothered to recompile the later code.) 

Anyhow the existing binary library for macOs probably does not work for the arm64 case...

Reply:
@vadi2 added build instructions to the top comment.

@SlySven you're absolutely right. The arm64 build won't work with an x86_64 library. I did initially start with the head of https://github.com/discord/discord-rpc but ran into the exact issue you mention - the `DiscordRichPresence` struct is different between versions. So I went back to v3.3.0 and built from there. 
As you said, it doesn't appear that `discord-rpc` or it's replacement are supported or maintained right now.

Reply:
> However you haven't clarified the point - does Discord provide an application that runs on the MacOS arm64 platform that the Discord-RPC library could connect to?

My apologies, I was focused on achieving binary compatibility and didn't fully understand what you were asking. The answer: Yes. I downloaded the official Discord client (arm64/x86_64 universal binary) and confirmed that the arm64 build here does connect locally and shows "Playing Mudlet" in my profile.